pom.xml

```diff
@@ -2188,11 +2195,26 @@
       </activation>
       <properties>
         <hbase.profile>2.6</hbase.profile>
-        <!-- Same compat version as 2.6.0, but different Hadoop version -->
-        <hbase.compat.version>2.6.0</hbase.compat.version>
+        <hbase.compat.version>2.6.4</hbase.compat.version>
         <hbase.version>${hbase-2.6.runtime.version}</hbase.version>
       </properties>
     </profile>
+    <profile>
+      <!-- This WILL work with the public -hadoop3 artifacts -->
+      <id>phoenix-hbase-compat-2.6.3</id>
+      <activation>
+        <property>
+          <name>hbase.profile</name>
+          <value>2.6.3</value>
+        </property>
+      </activation>
+      <properties>
+        <hbase.profile>2.6.3</hbase.profile>
+        <hadoop.version>3.4.1</hadoop.version>
```

Comment:
Why did you add this ?
While 2.6.3 is built with 3.4.1, in my tests it ran fine with 3.4.2.
We generally want the latest Hadoop that the Hbase version works with (and can run the tests)